docs/guide/snaps-introduction.md

```diff
@@ -0,0 +1,69 @@
+# Snaps Tutorial
```

Comment:
This does not seem like a tutorial. Is this a mistake?

Reply:
Since this page is named "Snaps Introduction" and there is a subheader below called "Snaps Tutorial", this should probably be:

```suggestion
# Snaps Introduction
```